node/pkg/accountant/ntt_config.go

```diff
@@ -31,6 +31,9 @@ func nttGetEmitters(env common.Environment) (validEmitters, validEmitters, error
 			// Lido wstETH + NTT deployment emitters
 			{chainId: vaa.ChainIDEthereum, addr: "000000000000000000000000A1ACC1e6edaB281Febd91E3515093F1DE81F25c0"},
 			{chainId: vaa.ChainIDBSC, addr: "000000000000000000000000be3F7e06872E0dF6CD7FF35B7aa4Bb1446DC9986"},
+			// mUSD NTT deployment emitters
+			{chainId: vaa.ChainIDEthereum, addr: "00000000000000000000000056E27f1A8425515FFD4BD76A254Ac1a5c0B66D71"},
```

Comment:
Should these show up in the explorers? Because I don't see them.

Reply:
Mezo Transceiver: https://explorer.mezo.org/address/0x56E27f1A8425515FFD4BD76A254Ac1a5c0B66D71

Ethereum Transceiver: https://etherscan.io/address/0x76ddb3f1dde02391ef0a28664499b74c29d18d3e